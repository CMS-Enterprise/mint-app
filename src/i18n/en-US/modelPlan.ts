const modelPlan = {
  home: 'Home',
  teamRoles: {
    modelLead: 'Model Lead',
    leadership: 'Leadership',
    evaluation: 'Evaluation',
    learning: 'Learning',
    modelTeam: 'Model Team',
    itLead: 'IT Lead',
    quality: 'Quality'
  },
  planStatuses: {
    planDraft: 'Draft model plan',
    planComplete: 'Model plan complete',
    icipComplete: 'ICIP complete',
    cmmiClearance: 'Internal (CMMI) clearance',
    cmsClearance: 'CMS clearance',
    hhsClearance: 'HHS clearance',
    ombASRFClearance: 'OMB/ASRF clearance',
    cleared: 'Cleared',
    announced: 'Announced',
    paused: 'Paused',
    canceled: 'Canceled'
  },
  status: {
    heading: 'Update status',
    copy:
      'After you’ve iterated on your Model Plan, update the status so others know what stage it’s at in the design and clearance process.',
    label: 'What is the status of your Model Plan?',
    updateButton: 'Update status',
    return: 'Don’t update status and return to task list'
  },
  favorite: {
    modelLead: 'Model lead(s)',
    startDate: 'Start date',
    cRTDLs: 'CRs and TDLs',
    toBeDetermined: 'To be determined',
    noneEntered: 'None entered',
    follow: 'Follow',
    following: 'Following',
    success:
      'Success! You are no longer following {{-requestName}} and will not receive email notifications about this model.',
    failure:
      'There was an error unfollowing {{-requestName}}. Please try unfollowing the model again if you no longer wish to receive email notifications about it.',
    error: 'Failed to locate and unfollow model plan.'
  }
};

export default modelPlan;
