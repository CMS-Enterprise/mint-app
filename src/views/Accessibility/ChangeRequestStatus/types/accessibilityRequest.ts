/* tslint:disable */
/* eslint-disable */
// @generated
// This file was automatically generated and should not be edited.

import { AccessibilityRequestStatus } from "./../../../../types/graphql-global-types";

// ====================================================
// GraphQL query operation: accessibilityRequest
// ====================================================

export interface accessibilityRequest_accessibilityRequest_statusRecord {
  __typename: "AccessibilityRequestStatusRecord";
  status: AccessibilityRequestStatus;
}

export interface accessibilityRequest_accessibilityRequest {
  __typename: "AccessibilityRequest";
  id: UUID;
  name: string;
  statusRecord: accessibilityRequest_accessibilityRequest_statusRecord;
}

export interface accessibilityRequest {
  accessibilityRequest: accessibilityRequest_accessibilityRequest | null;
}

export interface accessibilityRequestVariables {
  id: UUID;
}
