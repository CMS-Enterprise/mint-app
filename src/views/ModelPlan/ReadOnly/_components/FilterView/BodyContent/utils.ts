const allPossibleFilterViews = {
  mdm: {
    basics: ['nameHistory'],
    beneficiaries: [
      'beneficiaries',
      'beneficiariesOther',
      'beneficiariesNote',
      'numberPeopleImpacted',
      'estimateConfidence',
      'beneficiaryOverlap',
      'beneficiaryOverlapNote',
      'precedenceRules'
    ]
  },
  filler: {
    basics: ['nameHistory']
  }
};

export default allPossibleFilterViews;
