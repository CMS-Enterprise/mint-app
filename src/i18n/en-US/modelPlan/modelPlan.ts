import { ModelViewFilter } from 'gql/gen/graphql';

import { TranslationModelPlan } from 'types/translation';

export const modelPlan: TranslationModelPlan = {
  modelName: {
    gqlField: 'modelName',
    goField: 'ModelName',
    dbField: 'model_name',
    label: 'Model name',
    dataType: 'string',
    formType: 'text'
  },
  previousName: {
    gqlField: 'previousName',
    goField: 'PreviousName',
    dbField: 'previous_name',
    label: 'Previous names',
    dataType: 'string',
    formType: 'text'
  },
  nameHistory: {
    gqlField: 'nameHistory',
    goField: 'NameHistory',
    dbField: 'name_history',
    label: 'Previous names',
    dataType: 'string',
    isArray: true,
    formType: 'text',
    filterGroups: [
      ModelViewFilter.OACT,
      ModelViewFilter.DFSDM,
      ModelViewFilter.CCW,
      ModelViewFilter.IPC,
      ModelViewFilter.IDDOC,
      ModelViewFilter.PBG,
      ModelViewFilter.MDM,
      ModelViewFilter.CBOSC
    ]
  },
  abbreviation: {
    gqlField: 'abbreviation',
    goField: 'Abbreviation',
    dbField: 'abbreviation',
    label: 'Short name',
    sublabel:
      'The abbreviation, acronym, or other common name used for the model.',
    dataType: 'string',
    formType: 'text'
  },
  archived: {
    gqlField: 'archived',
    goField: 'Archived',
    dbField: 'archived',
    label: 'Archived',
    dataType: 'boolean',
    formType: 'radio',
    options: {
      true: 'Yes',
      false: 'No'
    }
  },
  status: {
    gqlField: 'status',
    goField: 'Status',
    dbField: 'status',
    label: 'What is the status of your Model Plan?',
    dataType: 'enum',
    formType: 'select',
    options: {
      PLAN_DRAFT: 'Draft Model Plan',
      PLAN_COMPLETE: 'Model Plan complete',
      ICIP_COMPLETE: 'ICIP complete',
      INTERNAL_CMMI_CLEARANCE: 'In internal (CMMI) clearance',
      CMS_CLEARANCE: 'In CMS clearance',
      HHS_CLEARANCE: 'In HHS clearance',
      OMB_ASRF_CLEARANCE: 'In OMB clearance',
      CLEARED: 'Cleared',
      ANNOUNCED: 'Announced',
      ACTIVE: 'Active',
      ENDED: 'Ended',
      PAUSED: 'Paused',
      CANCELED: 'Canceled'
    }
  }
};

export const modelPlanMisc: Record<string, string> = {
  heading: 'Model name',
  modelID: 'Model ID',
  createdBy: 'Created by',
  createdAt: 'Created at',
  readyForReviewBy: 'Ready for review by',
  readyForReviewAt: 'Ready for review at',
  breadcrumb: 'Start a new Model Plan',
  modeName: 'What is the name of your model?',
  modelNameInfo:
    'This is not a permanent name. If needed, you may update it later.',
  headingStatus: 'Update status',
  copy:
    'After you’ve iterated on your Model Plan, update the status so others know what stage it’s at in the design and clearance process.',
  updateButton: 'Update status',
  return: 'Don’t update status and return to task list'
};

export default modelPlan;
