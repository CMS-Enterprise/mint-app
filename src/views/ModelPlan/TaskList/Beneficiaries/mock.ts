import GetModelPlanBeneficiaries from 'queries/GetModelPlanBeneficiaries';
import { GetModelPlanBeneficiaries_modelPlan_beneficiaries as ModelPlanBeneficiariesType } from 'queries/types/GetModelPlanBeneficiaries';
import {
  BeneficiariesType,
  ConfidenceType,
  FrequencyType,
  OverlapType,
  SelectionMethodType,
  TaskStatus,
  TriStateAnswer
} from 'types/graphql-global-types';

export const beneficiaryMockData: ModelPlanBeneficiariesType = {
  __typename: 'PlanBeneficiaries',
  id: '123',
  modelPlanID: 'ce3405a0-3399-4e3a-88d7-3cfc613d2905',
  beneficiaries: ['OTHER' as BeneficiariesType],
  beneficiariesOther: 'Other Note',
  beneficiariesNote: '',
  treatDualElligibleDifferent: 'YES' as TriStateAnswer,
  treatDualElligibleDifferentHow: 'Differ text',
  treatDualElligibleDifferentNote: '',
  excludeCertainCharacteristics: 'YES' as TriStateAnswer,
  excludeCertainCharacteristicsCriteria: '',
  excludeCertainCharacteristicsNote: '',
  numberPeopleImpacted: 100,
  estimateConfidence: 'COMPLETELY' as ConfidenceType,
  confidenceNote: '',
  beneficiarySelectionNote: '',
  beneficiarySelectionOther: '',
  beneficiarySelectionMethod: ['HISTORICAL' as SelectionMethodType],
  beneficiarySelectionFrequency: 'ANNUALLY' as FrequencyType,
  beneficiarySelectionFrequencyNote: '',
  beneficiarySelectionFrequencyOther: '',
  beneficiaryOverlap: 'YES_NO_ISSUES' as OverlapType,
  beneficiaryOverlapNote: '',
  precedenceRules: '',
  createdBy: '',
  createdDts: '',
  modifiedBy: '',
  modifiedDts: '',
  status: TaskStatus.COMPLETE
};

const beneficiaryMock = [
  {
    request: {
      query: GetModelPlanBeneficiaries,
      variables: { id: 'ce3405a0-3399-4e3a-88d7-3cfc613d2905' }
    },
    result: {
      data: {
        modelPlan: {
          id: '1029',
          modelName: 'My excellent plan that I just initiated',
          generalCharacteristics: beneficiaryMockData
        }
      }
    }
  }
];

export default beneficiaryMock;
