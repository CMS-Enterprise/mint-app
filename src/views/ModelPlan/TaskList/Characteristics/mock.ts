import GetModelPlanCharacteristics from 'queries/GetModelPlanCharacteristics';
import { GetModelPlanCharacteristics_modelPlan_generalCharacteristics as ModelPlanCharacteristicsType } from 'queries/types/GetModelPlanCharacteristics';
import {
  GeographyType,
  KeyCharacteristic,
  TaskStatus
} from 'types/graphql-global-types';

export const charactersticMockData: ModelPlanCharacteristicsType = {
  __typename: 'PlanGeneralCharacteristics',
  id: '123',
  modelPlanID: 'ce3405a0-3399-4e3a-88d7-3cfc613d2905',
  isNewModel: false,
  existingModel: 'Second Plan',
  resemblesExistingModel: false,
  resemblesExistingModelWhich: [],
  resemblesExistingModelHow: '',
  resemblesExistingModelNote: '',
  hasComponentsOrTracks: true,
  hasComponentsOrTracksDiffer: 'Differ text',
  hasComponentsOrTracksNote: 'Component note',
  alternativePaymentModel: false,
  alternativePaymentModelTypes: [],
  alternativePaymentModelNote: '',
  keyCharacteristics: ['OTHER' as KeyCharacteristic],
  keyCharacteristicsOther: 'Key other note',
  keyCharacteristicsNote: '',
  collectPlanBids: false,
  collectPlanBidsNote: '',
  managePartCDEnrollment: false,
  managePartCDEnrollmentNote: '',
  planContactUpdated: false,
  planContactUpdatedNote: '',
  careCoordinationInvolved: false,
  careCoordinationInvolvedDescription: '',
  careCoordinationInvolvedNote: '',
  additionalServicesInvolved: true,
  additionalServicesInvolvedDescription: 'Lots of additional services',
  additionalServicesInvolvedNote: '',
  communityPartnersInvolved: false,
  communityPartnersInvolvedDescription: '',
  communityPartnersInvolvedNote: '',
  geographiesTargeted: true,
  geographiesTargetedTypes: ['OTHER' as GeographyType],
  geographiesTargetedTypesOther: 'Other geography type',
  geographiesTargetedAppliedTo: [],
  geographiesTargetedAppliedToOther: '',
  geographiesTargetedNote: '',
  participationOptions: false,
  participationOptionsNote: '',
  agreementTypes: [],
  agreementTypesOther: '',
  multiplePatricipationAgreementsNeeded: false,
  multiplePatricipationAgreementsNeededNote: '',
  rulemakingRequired: true,
  rulemakingRequiredDescription: 'Yes rulemaking is required',
  rulemakingRequiredNote: '',
  authorityAllowances: [],
  authorityAllowancesOther: '',
  authorityAllowancesNote: '',
  waiversRequired: false,
  waiversRequiredTypes: [],
  waiversRequiredNote: '',
  createdBy: '',
  createdDts: '',
  modifiedBy: '',
  modifiedDts: '',
  status: TaskStatus.COMPLETE
};

const charactersticMock = [
  {
    request: {
      query: GetModelPlanCharacteristics,
      variables: { id: 'ce3405a0-3399-4e3a-88d7-3cfc613d2905' }
    },
    result: {
      data: {
        modelPlan: {
          modelName: 'My excellent plan that I just initiated',
          generalCharacteristics: charactersticMockData
        }
      }
    }
  }
];

export default charactersticMock;
