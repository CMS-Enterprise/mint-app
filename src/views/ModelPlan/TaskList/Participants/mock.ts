import GetModelPlanParticipantsAndProviders from 'queries/GetModelPlanParticipantsAndProviders';
import { GetModelPlanProvidersAndParticipants_modelPlan_participantsAndProviders as ModelPlanParticipantsAndProvidersType } from 'queries/types/GetModelPlanProvidersAndParticipants';
import {
  ParticipantRiskType,
  ParticipantsIDType,
  ParticipantsType,
  ProviderAddType,
  TaskStatus
} from 'types/graphql-global-types';

export const participantsAndProvidersData: ModelPlanParticipantsAndProvidersType = {
  __typename: 'PlanParticipantsAndProviders',
  id: '123',
  modelPlanID: 'ce3405a0-3399-4e3a-88d7-3cfc613d2905',
  participants: [
    'MEDICARE_PROVIDERS' as ParticipantsType,
    'STATES' as ParticipantsType,
    'OTHER' as ParticipantsType
  ],
  medicareProviderType: 'Oncology Providers',
  statesEngagement:
    'States will determine administration specific to the state',
  participantsOther: 'The candy people',
  participantsNote: '',
  participantsCurrentlyInModels: null,
  participantsCurrentlyInModelsNote: '',
  modelApplicationLevel: 'c92.00 and c92.01',
  expectedNumberOfParticipants: 350,
  estimateConfidence: null,
  confidenceNote: '',
  recruitmentMethod: null,
  recruitmentOther: '',
  recruitmentNote: '',
  selectionMethod: [],
  selectionOther: '',
  selectionNote: '',
  communicationMethod: [],
  communicationNote: '',
  participantAssumeRisk: true,
  riskType: 'OTHER' as ParticipantRiskType,
  riskOther: 'Programmatic Risk',
  riskNote: '',
  willRiskChange: null,
  willRiskChangeNote: '',
  coordinateWork: null,
  coordinateWorkNote: '',
  gainsharePayments: null,
  gainsharePaymentsTrack: null,
  gainsharePaymentsNote: '',
  participantsIds: ['OTHER' as ParticipantsIDType],
  participantsIdsOther: 'Candy Kingdom Operations Number',
  participantsIDSNote: '',
  providerAdditionFrequency: null,
  providerAdditionFrequencyOther: '',
  providerAdditionFrequencyNote: '',
  providerAddMethod: ['OTHER' as ProviderAddType],
  providerAddMethodOther: 'Competitive ball-room dancing, free for all',
  providerAddMethodNote: '',
  providerLeaveMethod: [],
  providerLeaveMethodOther: '',
  providerLeaveMethodNote: '',
  providerOverlap: null,
  providerOverlapHierarchy: '',
  providerOverlapNote: '',
  createdBy: '',
  createdDts: '',
  modifiedBy: '',
  modifiedDts: '',
  status: TaskStatus.COMPLETE
};

const participantsAndProvidersMock = [
  {
    request: {
      query: GetModelPlanParticipantsAndProviders,
      variables: { id: 'ce3405a0-3399-4e3a-88d7-3cfc613d2905' }
    },
    result: {
      data: {
        modelPlan: {
          modelName: 'My excellent plan that I just initiated',
          participantsAndProviders: participantsAndProvidersData
        }
      }
    }
  }
];

export default participantsAndProvidersMock;
