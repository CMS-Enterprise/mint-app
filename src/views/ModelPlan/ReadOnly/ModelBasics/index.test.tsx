import React from 'react';
import { MemoryRouter, Route } from 'react-router-dom';
import { MockedProvider } from '@apollo/client/testing';
import { render, screen, waitFor } from '@testing-library/react';
import { mount } from 'enzyme';
import toJson, { OutputMapper } from 'enzyme-to-json';

import { modelBasicsMocks as mocks, modelID } from 'data/mock/readonly';
import { ModelCategory } from 'types/graphql-global-types';
import { translateModelCategory } from 'utils/modelPlan';
import renameTooltipAriaAndID from 'utils/testing/snapshotSerializeReplacements';

import ReadOnlyModelBasics from './index';

describe('Read Only Model Plan Summary -- Model Basics', () => {
  it('renders without errors', async () => {
    render(
      <MemoryRouter
        initialEntries={[`/models/${modelID}/read-only/model-basics`]}
      >
        <MockedProvider mocks={mocks} addTypename={false}>
          <Route path="/models/:modelID/read-only/model-basics">
            <ReadOnlyModelBasics modelID={modelID} />
          </Route>
        </MockedProvider>
      </MemoryRouter>
    );

    await waitFor(() => {
      expect(
        screen.getByTestId('read-only-model-plan--model-basics')
      ).toBeInTheDocument();
      expect(screen.getByText('Second Name')).toBeInTheDocument();
      expect(
        screen.getByText(translateModelCategory(ModelCategory.STATE_BASED))
      ).toBeInTheDocument();
    });
  });
  it('matches snapshot', async () => {
    const component = mount(
      <MemoryRouter
        initialEntries={[`/models/${modelID}/read-only/model-basics`]}
      >
        <MockedProvider mocks={mocks} addTypename={false}>
          <Route path="/models/:modelID/read-only/model-basics">
            <ReadOnlyModelBasics modelID={modelID} />
          </Route>
        </MockedProvider>
      </MemoryRouter>
    );

    await waitFor(() => {
      expect(component.text().includes('The Center for Awesomeness')).toBe(
        true
      );
    });

    expect(
      toJson(component, {
        mode: 'deep',
        map: renameTooltipAriaAndID as OutputMapper
      })
    ).toMatchSnapshot();
  });
});
