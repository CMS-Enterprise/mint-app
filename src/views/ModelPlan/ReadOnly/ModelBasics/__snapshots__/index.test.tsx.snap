// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Read Only Model Plan Summary -- Model Basics matches snapshot 1`] = `
<div
  className="read-only-model-plan--model-basics"
  data-testid="read-only-model-plan--model-basics"
>
  <div
    className="display-flex flex-justify flex-align-start"
  >
    <h2
      className="margin-top-0 margin-bottom-4"
    >
      Model basics
    </h2>
    <div />
  </div>
  <div
    className="desktop:grid-col-12"
    data-testid="grid"
  >
    <div
      className="read-only-section read-only-section--previous-model-names margin-bottom-3"
    >
      <p
        className="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
      >
        Previous model names
      </p>
      <p
        className="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
      >
        <em
          className="text-base"
        >
          No answer entered
        </em>
      </p>
    </div>
  </div>
  <div
    className="bg-base-lightest padding-2 margin-top-4 margin-bottom-4"
  >
    <p
      className="margin-top-0 text-bold"
    >
      Other identifiers
    </p>
    <p
      className="line-height-mono-4"
    >
      These are created in 
      <span
        className="mint-no-print"
      >
        <a
          aria-label="Open AMS in a new tab"
          className="usa-link usa-link--external"
          href="https://ams.cmmi.cms.gov"
          rel="noopener noreferrer"
          target="_blank"
        >
          CMMI Analysis & Management System (AMS).
        </a>
      </span>
      <span
        className="mint-only-print-inline"
      >
        CMMI Analysis & Management System (AMS).
      </span>
       Skip these fields until your model has been added to AMS. Not all models will have demo codes.
    </p>
    <div
      className="grid-row grid-gap"
      data-testid="grid"
    >
      <div
        className="desktop:grid-col-6"
        data-testid="grid"
      >
        <p
          className="text-bold margin-top-0 margin-bottom-1"
        >
          Model ID
        </p>
        <div
          className="text-italic text-base"
        >
          None entered
        </div>
      </div>
      <div
        className="desktop:grid-col-6"
        data-testid="grid"
      >
        <p
          className="text-bold margin-top-0 margin-bottom-1"
        >
          Demo code(s)
        </p>
        <div
          className="text-italic text-base"
        >
          None entered
        </div>
      </div>
    </div>
  </div>
  <div
    className="grid-row grid-gap"
    data-testid="grid"
  >
    <div
      className="desktop:grid-col-6"
      data-testid="grid"
    >
      <div
        className="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          className="read-only-section read-only-section--primary-model-category margin-bottom-3"
        >
          <p
            className="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            Primary model category
          </p>
          <div
            className="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
          >
            <span
              className="display-flex flex-align-center"
              style={
                Object {
                  "gap": "4px",
                }
              }
            >
              <span
                className="usa-tooltip"
                data-testid="tooltipWrapper"
              >
                <div
                  aria-describedby="tooltip"
                  className="usa-tooltip__trigger"
                  data-testid="triggerElement"
                  id="mint-tooltip"
                  onBlur={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onMouseOver={[Function]}
                  position="right"
                  tabIndex={0}
                  title=""
                >
                  <svg
                    className="usa-icon text-base-light"
                    focusable={false}
                    height="1em"
                    role="img"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                    <path
                      d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"
                    />
                  </svg>
                </div>
                <span
                  aria-describedby="mint-tooltip"
                  aria-hidden={true}
                  className="usa-tooltip__body"
                  data-testid="tooltipBody"
                  id="mint-tooltip-body"
                  role="tooltip"
                  style={Object {}}
                  title="modelCategory.tooltip.undefined"
                >
                  modelCategory.tooltip.undefined
                </span>
              </span>
            </span>
          </div>
        </div>
      </div>
    </div>
    <div
      className="desktop:grid-col-6"
      data-testid="grid"
    >
      <div
        className="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          className="read-only-section read-only-section--additional-model-categories margin-bottom-3"
        >
          <p
            className="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            Additional model categories
          </p>
          <p
            className="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
          >
            <em
              className="text-base"
            >
              No answer entered
            </em>
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    className="grid-row grid-gap"
    data-testid="grid"
  >
    <div
      className="desktop:grid-col-6"
      data-testid="grid"
    >
      <div
        className="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          className="read-only-section read-only-section--cms-component margin-bottom-3"
        >
          <p
            className="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            CMS component
          </p>
          <p
            className="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
          >
            <em
              className="text-base"
            >
              No answer entered
            </em>
          </p>
        </div>
      </div>
    </div>
    <div
      className="desktop:grid-col-6"
      data-testid="grid"
    >
      <div
        className="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          className="read-only-section read-only-section--cmmi-group margin-bottom-3"
        >
          <p
            className="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            CMMI Group
          </p>
          <p
            className="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
          >
            <em
              className="text-base"
            >
              No answer entered
            </em>
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    className="desktop:grid-col-12"
    data-testid="grid"
  >
    <div
      className="read-only-section read-only-section--model-type margin-bottom-3"
    >
      <p
        className="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
      >
        Model Type
      </p>
      <p
        className="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
      >
        <em
          className="text-base"
        >
          No answer entered
        </em>
      </p>
    </div>
  </div>
  <div
    className="desktop:grid-col-12"
    data-testid="grid"
  >
    <div
      className="read-only-section read-only-section--problem-statement margin-bottom-3"
    >
      <p
        className="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
      >
        Problem statement
      </p>
      <p
        className="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
      >
        <em
          className="text-base"
        >
          No answer entered
        </em>
      </p>
    </div>
  </div>
  <div
    className="desktop:grid-col-12"
    data-testid="grid"
  >
    <div
      className="read-only-section read-only-section--goal margin-bottom-3"
    >
      <p
        className="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
      >
        Goal
      </p>
      <p
        className="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
      >
        <em
          className="text-base"
        >
          No answer entered
        </em>
      </p>
    </div>
  </div>
  <div
    className="desktop:grid-col-12"
    data-testid="grid"
  >
    <div
      className="read-only-section read-only-section--test-interventions margin-bottom-3"
    >
      <p
        className="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
      >
        Test Interventions
      </p>
      <p
        className="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
      >
        <em
          className="text-base"
        >
          No answer entered
        </em>
      </p>
    </div>
  </div>
  <div
    className="mint-section read-only-model-plan__timeline--wrapper border-base-light padding-top-4  border-y-1px padding-bottom-2 margin-bottom-4 margin-top-6"
    data-testid="section-wrapper"
  >
    <h3
      className="margin-y-0"
    >
      Anticipated high level timeline
    </h3>
    <ol
      className="usa-process-list read-only-model-plan__timeline"
    >
      <li
        className="usa-process-list__item read-only-model-plan__timeline__list-item"
      >
        <p
          className="usa-process-list__heading font-body-sm line-height-sans-4"
          data-testid="processListHeading"
        >
          Complete ICIP
        </p>
        <p
          className="margin-y-0 font-body-md line-height-sans-4"
        >
          <em
            className="text-base"
          >
            No answer entered
          </em>
        </p>
      </li>
      <li
        className="usa-process-list__item read-only-model-plan__timeline__list-item"
      >
        <p
          className="usa-process-list__heading font-body-sm line-height-sans-4"
          data-testid="processListHeading"
        >
          Clearance
        </p>
        <div
          className="mobile-lg:display-flex"
        >
          <div
            className="width-card-lg margin-bottom-2 mobile-lg:margin-bottom-0"
          >
            <p
              className="usa-process-list__heading font-body-sm line-height-sans-4"
              data-testid="processListHeading"
            >
              Clearance start date
            </p>
            <p
              className="margin-y-0 font-body-md line-height-sans-4"
            >
              <em
                className="text-base"
              >
                No answer entered
              </em>
            </p>
          </div>
          <div
            className="width-card-lg margin-bottom-2 mobile-lg:margin-bottom-0"
          >
            <p
              className="usa-process-list__heading font-body-sm line-height-sans-4"
              data-testid="processListHeading"
            >
              Clearance end date
            </p>
            <p
              className="margin-y-0 font-body-md line-height-sans-4"
            >
              <em
                className="text-base"
              >
                No answer entered
              </em>
            </p>
          </div>
        </div>
      </li>
      <li
        className="usa-process-list__item read-only-model-plan__timeline__list-item"
      >
        <p
          className="usa-process-list__heading font-body-sm line-height-sans-4"
          data-testid="processListHeading"
        >
          Announce model
        </p>
        <p
          className="margin-y-0 font-body-md line-height-sans-4"
        >
          <em
            className="text-base"
          >
            No answer entered
          </em>
        </p>
      </li>
      <li
        className="usa-process-list__item read-only-model-plan__timeline__list-item"
      >
        <p
          className="usa-process-list__heading font-body-sm line-height-sans-4"
          data-testid="processListHeading"
        >
          Application period
        </p>
        <div
          className="mobile-lg:display-flex"
        >
          <div
            className="width-card-lg margin-bottom-2 mobile-lg:margin-bottom-0"
          >
            <p
              className="usa-process-list__heading font-body-sm line-height-sans-4"
              data-testid="processListHeading"
            >
              Application start date
            </p>
            <p
              className="margin-y-0 font-body-md line-height-sans-4"
            >
              <em
                className="text-base"
              >
                No answer entered
              </em>
            </p>
          </div>
          <div
            className="width-card-lg margin-bottom-2 mobile-lg:margin-bottom-0"
          >
            <p
              className="usa-process-list__heading font-body-sm line-height-sans-4"
              data-testid="processListHeading"
            >
              Application end date
            </p>
            <p
              className="margin-y-0 font-body-md line-height-sans-4"
            >
              <em
                className="text-base"
              >
                No answer entered
              </em>
            </p>
          </div>
        </div>
      </li>
      <li
        className="usa-process-list__item read-only-model-plan__timeline__list-item"
      >
        <p
          className="usa-process-list__heading font-body-sm line-height-sans-4"
          data-testid="processListHeading"
        >
          Performance period
        </p>
        <div
          className="mobile-lg:display-flex"
        >
          <div
            className="width-card-lg margin-bottom-2 mobile-lg:margin-bottom-0"
          >
            <p
              className="usa-process-list__heading font-body-sm line-height-sans-4"
              data-testid="processListHeading"
            >
              Performance start date
            </p>
            <p
              className="margin-y-0 font-body-md line-height-sans-4"
            >
              <em
                className="text-base"
              >
                No answer entered
              </em>
            </p>
          </div>
          <div
            className="width-card-lg margin-bottom-2 mobile-lg:margin-bottom-0"
          >
            <p
              className="usa-process-list__heading font-body-sm line-height-sans-4"
              data-testid="processListHeading"
            >
              Performance end date
            </p>
            <p
              className="margin-y-0 font-body-md line-height-sans-4"
            >
              <em
                className="text-base"
              >
                No answer entered
              </em>
            </p>
          </div>
        </div>
      </li>
      <li
        className="usa-process-list__item read-only-model-plan__timeline__list-item"
      >
        <p
          className="usa-process-list__heading font-body-sm line-height-sans-4"
          data-testid="processListHeading"
        >
          Model wrap-up end date
        </p>
        <p
          className="margin-y-0 font-body-md line-height-sans-4"
        >
          <em
            className="text-base"
          >
            No answer entered
          </em>
        </p>
      </li>
    </ol>
  </div>
  <div
    className="desktop:grid-col-12"
    data-testid="grid"
  >
    <div
      className="read-only-section read-only-section--if-timelines-are-tight--might-there-be-pieces-of-the-model-that-can-be-phased-in-over-time margin-bottom-3"
    >
      <p
        className="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
      >
        If timelines are tight, might there be pieces of the model that can be phased in over time?
      </p>
      <p
        className="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
      >
        <em
          className="text-base"
        >
          No answer entered
        </em>
      </p>
    </div>
  </div>
</div>
`;
